legend-depot-store-metrics/src/main/java/org/finos/legend/depot/store/metrics/services/QueryMetricsHandler.java

```diff
@@ -69,4 +72,22 @@ public List<VersionQueryMetric> getSummaryByProjectVersion()
         });
         return all.values().stream().sorted(MOST_RECENTLY_QUERIED).collect(Collectors.toList());
     }
+
+    public void consolidateMetrics()
+    {
+        List<VersionQueryMetric> versionQueryMetric = getSummaryByProjectVersion();
+        LOGGER.info("Started consolidating metrics for all project versions");
+        versionQueryMetric.forEach(metric ->
+        {
+            try
+            {
+                metricsStore.consolidate(metric);
+            }
+            catch (Exception e)
+            {
+                LOGGER.error(String.format("Error consolidating metrics for %s-%s-%s with error: %s", metric.getGroupId(), metric.getArtifactId(), metric.getVersionId(), e.getMessage()));
```

Comment:
why are you simply swallogin the exception... how are we going to know it failed?